### ISSUE TITLE
RND parser categorizes as a CAT_LITERAL in ULINE length statement

### ISSUE THREAD

**Issue by alexlukas** [SAP/abap-cleaner#80]
I found the following error when cleaning up some legacy code of ours:

![image](https://github.com/SAP/abap-cleaner/assets/5578562/6f428f57-cb65-477c-ab5f-d073aaec8d64)

The error log shows the same information:

`20230727_144425 , line 5: Parse error in line 5: RND Parser categorizes '65' as a CAT_LITERAL`

The problem occurs when I try to clean the following snippet, referencing the length 65 of the ULINE command:

```
REPORT zi_kw_al_test.

START-OF-SELECTION.

  ULINE AT /(65).
```

We currently have version 1.4.0 installed, but I did not see a reference to this issue in the release notes.

**Comment by jmgrassau** [comment]
Hi Alex, 

thanks very much for reporting this! This is indeed a bug, because apparently, the ABAP cleaner parser does not properly handle ```/(65)``` yet. Same for other possible syntax such as ```/10(65)```, ```/(*)```, ```/col(**)``` etc. Will look into this!

Kind regards,
Jörg-Michael

P.S.: and you're right, this hasn't been fixed up to the current version!

**Comment by jmgrassau** [comment]
Hi Alex, 

this issue is now fixed for the next release. 

I also improved the display of all those potential cases, so numeric literals, identifiers, token operators etc. are now colored consistently:

![image](https://github.com/SAP/abap-cleaner/assets/79639298/d3b71840-afb8-485f-a43d-151a153f6256)

Kind regards,
Jörg-Michael

**Comment by alexlukas** [comment]
Thanks for your quick answer and fix!

**Comment by jmgrassau** [comment]
Hi Alex, 

this issue should now be fixed with version 1.5.1 which was just released!

Kind regards,
Jörg-Michael
